### ISSUE TITLE
Agree description field, submission msg, error msg are multiline but no breaks are rendered.

### ISSUE THREAD

**Issue by TomDeSmet** [verbb/formie#219]
You can add new lines inside these fields, but no `<br>` tags are set when rendered.

**Comment by engram-design** [comment]
Should be fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.2.28",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.3.0](https://github.com/verbb/formie/releases/tag/1.3.0)
